rs/execution_environment/tests/dts.rs

```diff
@@ -1126,6 +1126,12 @@ fn dts_aborted_execution_does_not_block_subnet_messages() {
                 let args = CanisterInfoRequest::new(aborted_canister_id, None).encode();
                 (method, call_args().other_side(args))
             }),
+            Method::CanisterMetadata => test_supported(|aborted_canister_id| {
+                let args =
+                    CanisterMetadataRequest::new(aborted_canister_id, "git_commit_id".to_string())
```

Comment:
What's the reasoning behind using `"git_commit_id"` here? Is it an arbitrary string? If so, could you please leave a comment?